neural_lam/datastore/multizarr/create_normalization_stats.py

```diff
@@ -0,0 +1,127 @@
+
+
+def create_normalization_stats_zarr(
+    data_config_path: str,
+    zarr_path: str = None,
+):
+    """
+    Compute mean and std.-dev. for state and forcing variables and save them to
+    a Zarr file.
+
+    Parameters
+    ----------
+    data_config_path : str
+        Path to data config file.
+    zarr_path : str, optional
+        Path to save the normalization statistics to. If not provided, the
+        statistics are saved to the same directory as the data config file with
+        the name `normalization.zarr`.
+
+    """
+    if zarr_path is None:
+        zarr_path = Path(data_config_path).parent / DEFAULT_FILENAME
+
+    datastore = MultiZarrDatastore(config_path=data_config_path)
+
+    da_state = datastore.get_dataarray(category="state", split="train")
+    da_forcing = datastore.get_dataarray(category="forcing", split="train")
+
+    print("Computing mean and std.-dev. for parameters...", flush=True)
+    da_state_mean, da_state_std = compute_stats(da_state)
+
+    if da_forcing is not None:
+        da_forcing_mean, da_forcing_std = compute_stats(da_forcing)
+        combined_stats = datastore._config["utilities"]["normalization"][
+            "combined_stats"
+        ]
+
+        if combined_stats is not None:
+            for group in combined_stats:
+                vars_to_combine = group["vars"]
+
+                da_forcing_means = da_forcing_mean.sel(
+                    forcing_feature=vars_to_combine
+                )
+                stds = da_forcing_std.sel(forcing_feature=vars_to_combine)
+
+                combined_mean = da_forcing_means.mean(dim="forcing_feature")
+                combined_std = (stds**2).mean(dim="forcing_feature") ** 0.5
+
+                da_forcing_mean.loc[
+                    dict(forcing_feature=vars_to_combine)
+                ] = combined_mean
+                da_forcing_std.loc[
+                    dict(forcing_feature=vars_to_combine)
+                ] = combined_std
+    print(
+        "Computing mean and std.-dev. for one-step differences...", flush=True
+    )
+    state_data_normalized = (da_state - da_state_mean) / da_state_std
+    state_data_diff_normalized = state_data_normalized.diff(dim="time")
+    diff_mean, diff_std = compute_stats(state_data_diff_normalized)
+
+    ds = xr.Dataset(
+        {
+            "state_mean": da_state_mean,
+            "state_std": da_state_std,
+            "state_diff_mean": diff_mean,
+            "state_diff_std": diff_std,
+        }
+    )
+
+    if da_forcing is not None:
+        dsf = xr.Dataset(
+            {
+                "forcing_mean": da_forcing_mean,
+                "forcing_std": da_forcing_std,
+            }
+        )
+        ds = xr.merge([ds, dsf])
+
+    ds = ds.chunk({"state_feature": -1, "forcing_feature": -1})
+    print("Saving dataset as Zarr...")
+    ds.to_zarr(zarr_path, mode="w")
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Training arguments",
```

Comment:
```suggestion
        description="Create standardization statistics",
```
(In general this file should use standardization instead of normalization.)